personal_finances/bank_interface/client.py

```diff
@@ -69,6 +79,29 @@ def __exit__(
         )
         LOGGER.info("uvicorn server killed")
 
+    def _handle_tokens(self) -> None:
+        self._token_store = DiskTokenStore()
+        if self._token_store.is_access_token_valid(TEMPORARY_FIXED_USER_ID):
+            LOGGER.info("valid access token found")
+            self._nordigen_client.token = self._token_store.get_last_token(
+                TEMPORARY_FIXED_USER_ID
+            ).AccessToken
+
+        elif self._token_store.is_refresh_token_valid(TEMPORARY_FIXED_USER_ID):
+            LOGGER.info("valid refresh token found")
+            self._nordigen_client.exchange_token(
+                self._token_store.get_last_token(TEMPORARY_FIXED_USER_ID).RefreshToken
+            )
```

Comment:
Should not save again the same token?

Reply:
You reminded me of asserting if `save_token` is called in other tests as well